gips/data/modis/modis.py

```diff
@@ -470,6 +470,14 @@ class modisData(Data):
             'bands': ['cloud-cover'],
             'startdate': datetime.date(2000, 2, 24),
             'latency': 3
+        },
+        'ref': {
+            'description': 'Surface reflectance',
+            'assets': [MCD43A4],
+            'sensor': 'MCD',
+            'bands': ['BLUE', 'GREEN', 'RED', 'NIR', 'SWIR1', 'SWIR2'],
+            'startdate': datetime.date(2000, 2, 18),
+            'latency': 15,
         }
```

Comment:
It's usually not worse than 10 days back.

Reply:
Changed to 5